src/payment/offsite-payment-mappers/payload-mapper.js

```diff
@@ -87,6 +87,16 @@ export default class PayloadMapper {
             this.storeMapper.mapToStore(data)
         );
 
+        const { formattedPayload = {} } = payment;
+        const { browserInfo = {} } = formattedPayload;
```

Comment:
to keep it consistent to how `formattedPayload` is used in other scenarios, the content should either be directly assigned to payload `Object.assign(payload, formattedPayload)`, or directly strip `browserInfo` from `payment`

```js
const { browserInfo = {} } = payment;
```